pkg/planner/core/memtable_predicate_extractor.go

```diff
@@ -1812,7 +1812,7 @@ func (e *InfoSchemaTablesExtractor) Extract(ctx base.PlanContext,
 	predicates []expression.Expression,
 ) (remained []expression.Expression) {
 	var resultSet, resultSet1 set.StringSet
-	e.colNames = []string{"table_schema", "table_name"}
+	e.colNames = []string{"table_schema", "constraint_schema", "table_name", "constraint_name", "sequence_schema", "sequence_name", "partition_name", "schema_name", "index_name"}
```

Comment:
the field `InfoSchemaTablesExtractor.colNames` can be replaced by a global constant-like variable?